test/sox_io_backend/test_info.py

```diff
@@ -0,0 +1,110 @@
+import itertools
+from parameterized import parameterized
+
+from torchaudio.backend import sox_io_backend
+
+from ..common_utils import (
+    TempDirMixin,
+    PytorchTestCase,
+    skipIfNoExec,
+    skipIfNoExtension,
+)
+from .common import (
+    get_test_name
+)
+from . import sox_utils
+
+
+@skipIfNoExec('sox')
+@skipIfNoExtension
+class TestInfo(TempDirMixin, PytorchTestCase):
+    @parameterized.expand(list(itertools.product(
+        ['float32', 'int32', 'int16', 'uint8'],
+        [8000, 16000],
+        [1, 2],
```

Comment:
nit: this is nice :) is there a way to have pass keyword arguments with parameterized?

Reply:
I do not know. Please refer to the doc https://pypi.org/project/parameterized/